full-service/src/json_rpc/v1/api/wallet.rs

```diff
@@ -117,6 +117,18 @@ where
 {
     global_log::info!("Running command {:?}", command);
 
+    if service.resync_in_progress().map_err(format_error)? {
+        let wallet_status = service.get_wallet_status().map_err(format_error)?;
+
+        let percent_complete = (wallet_status.min_synced_block_index as f64
+            / wallet_status.local_block_height as f64
+            * 100.0) as u64;
```

Comment:
You should be able to skip the floating point stuff by doing `wallet_status.min_synced_block_index * 100 / wallet_status.local_block_height`